Homework/js/homework-10.js

```diff
@@ -0,0 +1,133 @@
+// 1)
+const camelCase = (string) => {
+    const getStringWithSpace = string.split(/(?=[A-Z])/).join(" ");
+
+    return getStringWithSpace;
+}
+
+console.log(camelCase("helloMyDearFriend"));
+
+// 2)
+const findMissing = (numbers) => {
+    const maxValue = Math.max(...numbers);
+
+    for (let i = 1; i <= maxValue; i++) {
+        if (i % 2 !== 0 && !numbers.includes(i)) {
+            return i;
+        }
+    }
+
+    return "Nothing is found!"
+};
+
+console.log(findMissing([1, 3, 5, 9, 11]));
+
+
+// 3)
+const romanNumeral = {
+    I: 1,
+    V: 5,
+    X: 10,
+    L: 50,
+    C: 100,
+    D: 500,
+    M: 1000,
+};
+
+const getDecimalNumeral = (allNumeral, date) => {
```

Comment:
what is date?? There are not any date arg in task

Reply:
Corrected